### PR TITLE
Add an initiative for our ongoing docs push

### PR DESCRIPTION
**Pull request by decentralion**
We have a lot of good work happening in sourcecred/docs, and I felt it
would be a good opportunity for more experimental usage of our
initiatives system. I went through the history of the repo and added
major pull requests as contributions; my heuristic was that included
pull requests should either:
- contribute material new content to the documentation, or
- make material technical progress towards hosting and serving the docs

I did not include every pull request as some are duplicates (e.g. the
same content was placed in a few pull requests) and some didn't meet my
materiality threshold. When we add the ability to set
'contribution weights' we will be able to do this in a more fine-grained
way.

I also added several forum posts and issues as 'references'.

If you feel that some contributions should/should not be included, feel
free to say so, focusing on the value of the contribution in question.
Please keep in mind that [initiatives are still experimental][1].

I set the cred weight as 200 when incomplete, and 1000 when complete.
This feels roughly in line with our current cred totals (we may revise
this later).

[1]: https://discourse.sourcecred.io/t/rolling-out-initiatives/698

Test plan: Initiative loads without parse errors, and the resultant cred
is different from cred before (people who participated have slightly
more cred). It's not a huge effect because I set this initiative as
worth 200 cred while incomplete.

### CODE REVIEW

**Comment by decentralion** [comment]
cc folks who have been working on docs: @vsoch @s-ben @hammadj

File: initiatives/2020-04-docs.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      "https://github.com/sourcecred/docs/20",
+      "https://github.com/sourcecred/docs/19",
+      "https://github.com/sourcecred/docs/17",
+      "https://github.com/sourcecred/docs/13",
+      "https://github.com/sourcecred/docs/11",
+      "https://github.com/sourcecred/docs/10",
+      "https://github.com/sourcecred/docs/9",
+      "https://github.com/sourcecred/docs/2",
+      "https://github.com/sourcecred/docs/1"
+    ]
```
Comment (Beanow) [inline initiatives/2020-04-docs.json:44]: Seem like 404s. `/issues` or `/pulls` should be added before the number.
Does the reference detector actually resolve this? :thinking:
For now, the best check is to see what the legacy explorer provides.

File: initiatives/2020-04-docs.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    "references": [
+      "https://github.com/sourcecred/docs/issues/5",
+      "https://discourse.sourcecred.io/t/creating-documentation-on-github-review-culture/582",
+      "https://discourse.sourcecred.io/t/documentation-glossary-and-wishlist/354",
+      "https://discourse.sourcecred.io/t/weekly-community-call-1-28-2020/605",
+      "https://discourse.sourcecred.io/t/weekly-community-call-1-14-2019/578",
+      "https://discourse.sourcecred.io/t/weekly-community-call-1-7-2020/572"
+    ],
+    "contributions": [
+      "https://github.com/sourcecred/docs/47",
```
Comment (s-ben) [inline initiatives/2020-04-docs.json:27]: Also getting the 404s. It looks like the URL is missing the `pull/` part of the path. E.g. presumably this one is:
`https://github.com/sourcecred/docs/pull/47`

File: initiatives/2020-04-docs.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    "completed": false,
+    "champions": [
+    ],
+    "dependencies": [
+    ],
+    "references": [
+      "https://github.com/sourcecred/docs/issues/5",
+      "https://discourse.sourcecred.io/t/creating-documentation-on-github-review-culture/582",
+      "https://discourse.sourcecred.io/t/documentation-glossary-and-wishlist/354",
+      "https://discourse.sourcecred.io/t/weekly-community-call-1-28-2020/605",
```
Comment (s-ben) [inline initiatives/2020-04-docs.json:22]: While I appreciate the link to community call notes (which are generally documentation), presumably these are also going to be contributions to the community call Initiative I've been tasked to help flesh out? If so, I would say remove these. Call notes also just feel kinda like a different type of documentation...this feels more like an "official docs" initiative. 

More generally, while I feel I've made a few abstract, non-concrete docs contributions (e.g. playing docs "consultant" on calls, providing input to decisions), not sure how that would show up in this initiative. I have had the thought I would like to get more cred in the future by making my own PRs, which is a good thing IMO, as it shows the incentives are working. Alas, have been too busy to do anything other than review lately, which I presume will flow a bit of cred my way, via docs Initiative->PR->reviews/comments.

File: initiatives/2020-04-docs.json (hunk 1 above)
Comment (decentralion) [reply initiatives/2020-04-docs.json]: Thanks for catching this. Fixed.

File: initiatives/2020-04-docs.json (hunk 3 above)
Comment (decentralion) [reply initiatives/2020-04-docs.json:22]: I included them in references b.c. they were calls in which we discussed the documentation effort. But they definitely aren't as relevant as those posts/issues which were specifically oriented on the docs work. Removed per your suggestion.

**Comment by Beanow** [comment]
Going back to this. I think we've refined the scope of the Docs work a lot.
The way it's formulated for beta.1:
- Docs scaffold
- Intro Docs
- Concept Docs
- Guide Docs
- Setup & Usage Docs
...

I think would make it a lot more obvious what the Initiatives represent.

**Pull request closed without merging**